tasks/nginx.yml

```diff
@@ -3,7 +3,7 @@
 - name: Copy Nginx config for Jitsi Meet domain.
   template:
     dest: "/etc/nginx/sites-available/{{ jitsi_meet_server_name }}.conf"
-    src: jitsi_meet_nginx.conf.j2
+    src: "{{ jitsi_meet_nginx_conf }}"
```

Comment:
To maintain a unified criteria in vars names, here it would be better to call it `jitsi_meet_nginx_config_template`